src/js/react-timeslot-calendar.jsx

```diff
@@ -1,17 +1,35 @@
 import React from 'react';
-// import PropTypes from 'prop-types';
-import '../../styles/main.scss';
+import PropTypes from 'prop-types';
+
+import '../styles/main.scss';
+import Calendar from './components/calendar.jsx';
 
 export default class ReactTimeslotCalendar extends React.Component {
-  constructor(props) {
-    super(props);
-  }
 
   render() {
-    return null;
+    const {
+      initialDate,
+      timeslots,
+      selectedTimeslot,
+    } = this.props;
+
+    return (
+      <Calendar
+        initialDate = { initialDate }
+        timeslots = { timeslots }
+        selectedTimeslot = { selectedTimeslot }
+      />
+    );
   }
 }
 
```

Comment:
Since you've stated here that *timeslots* is required, it'd be good to add the *dReactTimeslotCalendar.defaultProps* and state that the *DEFAULT_TIMESLOTS* is default. Remember to import, of course.